decidim-budgets/lib/decidim/budgets/engine.rb

```diff
@@ -26,6 +26,11 @@ class Engine < ::Rails::Engine
       initializer "decidim_budgets.assets" do |app|
         app.config.assets.precompile += %w(decidim_budgets_manifest.js)
       end
+
+      initializer "decidim_budgets.add_cells_view_paths" do
+        Cell::ViewModel.view_paths << File.expand_path("#{Decidim::Budgets::Engine.root}/app/cells")
```

Comment:
At some point, we could make this implicit when registering a component's engines.